.github/workflows/commit_syntax.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Commit Message Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - '*'
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(Breaking|Update|Fix|New|Upgrade|No code):'
```

Comment:
Minor concern with keeping this in sync with our values in `package.json`. Not something that changes often though, so not too worried.

Reply:
Ideally, we could synchronize from a single data source, but yeah won't be a major or frequent disturbance.